esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -2139,6 +2139,16 @@ void CWsDfuEx::doGetFileDetails(IEspContext &context, IUserDescriptor *udesc, co
             }
             else if (isKeyFile)
                 FileDetails.setCompressedFileSize(size);
+
+            if (isKeyFile && (version >= 1.41))
+            {
+                if (df->queryAttributes().hasProp("@partitionFieldMask"))
```

Comment:
minor: in keeping with isFileKey() functions, it would be cleaner if there were equivalent for partition keys. e.g. isFilePartitionKey(df); 
Same for isFileLocalKey()